open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/test/resources/application.properties

```diff
@@ -0,0 +1,38 @@
+#SPDX-License-Identifier: Apache-2.0
```

Comment:
Gaian isn't mocked? or launched? So this can't work for most people? Or did I miss the mocking? ;-) More integration test oriented?

Reply:
It is not launched, there is only one test for now related to events pushed back to IV OMAS. These properties are set on the event(example gaian_schema, gaian_db) as they are needed by IV OMAS to  create the Endpoint, Connection and Information View. Build should work for everybody